tests/ensemble_evaluator/test_entity.py

```diff
@@ -83,70 +71,74 @@ def test_snapshot_merge():
         stage_id="0",
         step_id="0",
         job_id="0",
-        status="Finished",
-        start_time=datetime(year=2020, month=10, day=27).isoformat(),
-        end_time=datetime(year=2020, month=10, day=28).isoformat(),
-        data={"memory": 1000},
+        job=Job(
+            status="Finished",
+            start_time=datetime(year=2020, month=10, day=27).isoformat(),
+            end_time=datetime(year=2020, month=10, day=28).isoformat(),
+            data={"memory": 1000},
+        ),
     )
     update_event.update_job(
         real_id="1",
         stage_id="0",
         step_id="0",
         job_id="1",
-        status="Running",
-        start_time=datetime(year=2020, month=10, day=27).isoformat(),
+        job=Job(
+            status="Running",
+            start_time=datetime(year=2020, month=10, day=27).isoformat(),
+        ),
     )
     update_event.update_job(
         real_id="9",
         stage_id="0",
         step_id="0",
         job_id="0",
-        status="Running",
-        start_time=datetime(year=2020, month=10, day=27).isoformat(),
+        job=Job(
+            status="Running",
+            start_time=datetime(year=2020, month=10, day=27).isoformat(),
+        ),
     )
 
     snapshot.merge_event(update_event)
 
     assert snapshot.get_status() == "running"
 
-    assert _dict_equal(
-        snapshot.get_job(real_id="1", stage_id="0", step_id="0", job_id="0"),
-        {
-            "status": "Finished",
-            "start_time": "2020-10-27T00:00:00",
-            "end_time": "2020-10-28T00:00:00",
-            "data": {"memory": 1000},
-            "error": None,
-            "name": "job0",
-            "stderr": None,
-            "stdout": None,
-        },
+    assert snapshot.get_job(real_id="1", stage_id="0", step_id="0", job_id="0") == Job(
+        status="Finished",
+        start_time=dateutil.parser.isoparse("2020-10-27T00:00:00"),
```

Comment:
Why don't you use here `start_time=datetime(year=2020, month=10, day=27).isoformat()` as earlier (on line 88 for example)?

Reply:
And why you have `.isoformat()`? I think `Job` expects datetime object, does not it?

Reply:
isoparse gives a datetime object. But yeah, I can change it into just a datetime instantiation directly.

Reply:
Oh, you were probably referring to the other lines. Yeah, don't know why that worked, but I changed it now.